### ISSUE TITLE
Checkpoint of ResNet32 pre-trained using TinyImageNet

### ISSUE THREAD

**Issue by jnhwkim** [alinlab/L2T-ww#5]
Could I get a checkpoint of ~~VGG9~~ ResNet32 pre-trained on TinyImageNet in 
https://github.com/alinlab/L2T-ww/blob/master/train_l2t_ww.py#L177 used in the paper?

**Comment by hankook** [comment]
We used ResNet32 as a source model for 32x32-scale experiments. Thus, we have no VGG9 pre-trained on TinyImageNet. For pre-training ResNet32, we used the conventional optimization scheme, e.g., learning rate = 0.1 (with decaying by 0.1 at 50% and 75% epochs), batch size = 128, and so on. So you can easily obtain the pre-trained ResNet32.

I'm currently trying to find the checkpoint of ResNet32. I'll let you know if find it.

**Comment by jnhwkim** [comment]
My mistake. You're right for ResNet32. I would be grateful if the checkpoint is available for reproduction.

**Comment by hankook** [comment]
Sorry for late response. I recently found the [checkpoint](https://github.com/alinlab/L2T-ww/blob/master/logs/tinyimagenet-200-resnet32/0/model_best.pth.tar) of ResNet32 pre-trained on TinyImageNet. You can reproduce the CIFAR100/STL10 experiments using the following commands:
```sh
python train_l2t_ww.py --dataset cifar100 --datasplit cifar100 --dataroot /data/ --experiment logs/cifar100_0/ --source-path logs --source-model resnet32 --source-domain tinyimagenet-200 --target-model vgg9_bn --pairs 4-0,4-1,4-2,4-3,4-4,9-0,9-1,9-2,9-3,9-4,14-0,14-1,14-2,14-3,14-4 --batchSize 128
python train_l2t_ww.py --dataset stl10 --datasplit stl10 --dataroot /data/ --experiment logs/stl10_0/ --source-path logs --source-model resnet32 --source-domain tinyimagenet-200 --target-model vgg9_bn --pairs 4-0,4-1,4-2,4-3,4-4,9-0,9-1,9-2,9-3,9-4,14-0,14-1,14-2,14-3,14-4 --batchSize 128
```
When using the recent Pytorch (1.6.0), I obtained 71.55% and 78.60% test accuracy on CIFAR100 and STL10, respectively.
